### DISCUSSION TITLE
corfu preview candidate

### DISCUSSION THREAD

**Discussion by akhil3417** [minad/corfu#406] [Q&A] [answered]
![maim-region-20231229-210151](https://github.com/minad/corfu/assets/92385307/ee5858eb-6f88-4c36-92b8-3a588834b7bf)

have been trying to setup Codeium. and 
In the image above it does'nt preview the first suggestion , 

but below it does preview it , but not the whole thing 
![maim-region-20231229-210126](https://github.com/minad/corfu/assets/92385307/d31ee11e-2a74-4b2e-8113-dd41408aab3e)
any fixes ? 

anything that can mimic , `company-frontends '(company-pseudo-tooltip-frontend company-preview-frontend)`

**Comment by minad** [comment, marked as answer]
Duplicate #215. Please don't recreate issues. You can set `corfu-preselect` to `prompt`. Then the first candidate will be shown when selected. I guess the full candidate is not shown since it is a multi line candidate, and therefore hidden behind the popup. If a backend produces multi line candidates I suggest to set `corfu-preview-current` to nil.
